wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1257,24 +1302,25 @@ class SwiftGenerator private constructor(
     type: MessageType,
     storageName: String,
     storageType: DeclaredTypeName,
+    oneOfEnumNames: Map<OneOf, DeclaredTypeName>,
   ) {
     if (!type.isHeapAllocated) {
       println("Generating storage property delegates for a non-heap allocated type?!")
     }
 
     val propertyVariable = TypeVariableName.typeVariable("Property")
+    val signature = FunctionSignatureSpec.builder()
```

Comment:
Do we want/need this dynamic member lookup method anymore?

Reply:
Yup. We need it for `$foo`, even if `foo` explicitly exists.